zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -389,6 +389,20 @@ impl<'a, F: FieldExt> ConstraintBuilder<'a, F> {
         );
     }
 
+    // block
+    pub(crate) fn block_lookup(
+        &mut self,
+        tag: Expression<F>,
+        number_or_zero: Expression<F>,
```

Comment:
```suggestion
        number: Option<Expression<F>>,
```

Reply:
if change function parameter to `number` will result same name in Lookup:block field. such as   
```
self.add_lookup(Lookup::Block {
            field_tag: tag,
            number: number,
            value: val,
        });  
```
this led to clippy checking error.

Reply:
You need to change to
```rust
self.add_lookup(Lookup::Block { 
    field_tag: tag, 
    number: Some(number), 
    value: val, 
});
```